tests/assertions/ocp4/ocp4-cis-node-4.17.yml

```diff
@@ -0,0 +1,401 @@
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-ovs-pid:
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-ovs-sys-id-conf:
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-ovs-vswitchd-pid:
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-ovsdb-server-pid:
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-scheduler-kubeconfig:
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-worker-ca:
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-worker-kubeconfig:
+   default_result: PASS
+ e2e-cis-node-master-file-groupowner-worker-service:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-cni-conf:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-controller-manager-kubeconfig:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-etcd-data-dir:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-etcd-data-files:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-etcd-member:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-etcd-pki-cert-files:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ip-allocations:
+   default_result: NOT-APPLICABLE
+ e2e-cis-node-master-file-owner-kube-apiserver:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-kube-controller-manager:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-kube-scheduler:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-kubelet:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-kubelet-conf:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-master-admin-kubeconfigs:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-multus-conf:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-openshift-pki-cert-files:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-openshift-pki-key-files:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-openshift-sdn-cniserver-config:
+   default_result: NOT-APPLICABLE
+ e2e-cis-node-master-file-owner-ovn-cni-server-sock:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ovn-db-files:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ovs-conf-db:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ovs-conf-db-lock:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ovs-pid:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ovs-sys-id-conf:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ovs-vswitchd-pid:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-ovsdb-server-pid:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-scheduler-kubeconfig:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-worker-ca:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-worker-kubeconfig:
+   default_result: PASS
+ e2e-cis-node-master-file-owner-worker-service:
+   default_result: PASS
+ e2e-cis-node-master-file-permissions-cni-conf:
+   default_result: FAIL
```

Comment:
So this rule is passing on 4.17 for `cis-node`?

`     helpers.go:879: E2E-FAILURE: The expected result for the e2e-cis-node-master-file-permissions-cni-conf rule didn't match. Expected 'FAIL', Got 'PASS' `

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/ComplianceAsCode_content/12266/pull-ci-ComplianceAsCode-content-master-4.17-e2e-aws-ocp4-cis-node/1823364170288992256

All other profiles have this rule as `default_result: FAIL`.
I re-ran a `stig-node` to test...

Reply:
And current two failures are because of this rule, :grinning: 
```
     helpers.go:879: E2E-FAILURE: The expected result for the e2e-cis-node-worker-file-permissions-cni-conf rule didn't match. Expected 'FAIL', Got 'PASS' 
     helpers.go:879: E2E-FAILURE: The expected result for the e2e-cis-node-master-file-permissions-cni-conf rule didn't match. Expected 'FAIL', Got 'PASS' 
```

@rhmdnd Do you want to update those in this PR or in a separate one?

Reply:
I'll push a new version quick.